installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -42,6 +42,9 @@ include::modules/installing-ocp-agent-ibm-z-zvm.adoc[leveloffset=+2]
 // Adding {ibm-z-name} agents with {op-system-base} KVM
 include::modules/installing-ocp-agent-ibm-z-kvm.adoc[leveloffset=+2]
 
+// Adding {ibm-z-name} agents with {op-system-base} KVM for ISO boot
+include::modules/installing-ocp-agent-ibm-z-kvm-iso-boot.adoc[leveloffset=+2]
+
```

Comment:
You need to remove those lines again that's why the build fails

```suggestion
// Adding {ibm-z-name} agents with {op-system-base} KVM for ISO boot
include::modules/installing-ocp-agent-ibm-z-kvm-iso-boot.adoc[leveloffset=+2]

```

Reply:
+1 to this comment, this needs to be removed now that you included the other module with the ifdef statements.